QPM/Providers/RemoteQPMDependencyResolver.cs

```diff
@@ -455,6 +474,21 @@ public void ResolveDependency(in Config myConfig, in RestoredDependencyPair pair
             }
         }
 
+        private static void PlaceDep(SymLinker.Linker.Linker linker, string tempLoc, string fileLoc)
```

Comment:
using statement instead of full name, that is, if you still want this to be a function that takes it in as a parameter.